lambdex/bin/lambdex.py

```diff
@@ -202,11 +205,23 @@ def test_lambdex(args):
 
         with chdir(target):
             runner = EntryPoint.parse("run = %s" % lambdex_entry_point).resolve()
-            if args.empty:
-                runner({}, None)
-            else:
-                for filename in args.files:
-                    runner(load_json_blob(filename), None)
+            if args.type == EVENT_FUNCTION_SIGNATURE:
+                if args.empty:
+                    runner({}, None)
+                else:
+                    for filename in args.files:
+                        runner(load_json_blob(filename), None)
+            elif args.type == GCP_HTTP_FUNCTION_SIGNATURE:
+                import flask
```

Comment:
Would I put it under requires-extra or under `[project.optional-dependencies]`?

Reply:
No, that's only available in the PEP-621 support provided by https://flit.readthedocs.io/en/latest/pyproject_toml.html#new-style-metadata which is only supported by flit 3.2+. Pex must stay on old flit for now since Pex supports Python 2.7 and the pyproject.toml becomes part of the sdist and its build system when distributed to pypy. A Python 2.7 Pex user would then fail to be able to install Pex from sdist with Python 2.7.

Reply:
Sounds good, when with test-gcp-http for now and added it to integration test deps.